packages/core/base/src/services/hosted/crossmintPayButtonService.ts

```diff
@@ -1,18 +1,22 @@
-import { Locale, PaymentMethod } from "../../types";
+import { CheckoutProps, Locale, PaymentMethod } from "../../types";
 import { t } from "../../utils/i18n";
 
 interface IProps {
     onClick?: (e: any) => void;
     connecting: boolean;
     paymentMethod?: PaymentMethod;
     locale: Locale;
+    checkoutProps?: CheckoutProps;
 }
 
-export function crossmintPayButtonService({ onClick, connecting, paymentMethod, locale }: IProps) {
+export function crossmintPayButtonService({ onClick, connecting, paymentMethod, locale, checkoutProps }: IProps) {
     const getButtonText = (connecting: boolean) => {
         if (connecting) {
             return t("crossmintPayButtonService.CONNECTING", locale);
         }
+        if(checkoutProps?.experimental === true) {
```

Comment:
if a paymentMethod is passed, we probably want to include that in the button text

or if `paymentMethods` on has 1 item